vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -103,23 +102,13 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
           }
           vrrp_instance VI_1 {
               state MASTER
-              interface eth1
```

Comment:
You should not remove it , rename it perhaps

Reply:
rename to `enp1s0` | `enp2s0` | `eth0` ? Most probably `enp1s0`..

Reply:
You also need to keep the backwards compatibility for v1a3 without Metal3datatemplate

Reply:
Updated